docs/platforms/javascript/common/configuration/integrations/launchdarkly.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: LaunchDarkly
+description: "Attaches recent LaunchDarkly feature flag evaluations to error event context."
```

Comment:
```suggestion
description: "Learn how to attach LaunchDarkly feature flag evaluations to error event context."
```

Reply:
@cmanallen Should this be expanded to feature flag **change** tracking?

Reply:
Is this too verbose? "Learn how to attach recent LaunchDarkly feature flag evaluations to your error event context"

> @cmanallen Should this be expanded to feature flag change tracking?

@jas-kas The SDK integration doesn't track FF changes, only evaluations leading up to the error. Changes are tracked through the webhook, maybe we can link the docs for that

Reply:
^ basically +1 to Michelle's comment below!